tests/test_bibliotheca.py

```diff
@@ -1200,16 +1510,19 @@ def test_handle_event(self):
         assert 1 == pool.licenses_owned
         assert 1 == pool.licenses_available
 
-        # Four analytics events were collected: one for the license add
-        # event itself, one for the 'checkin' that made the new
-        # license available, one for the first appearance of a new
-        # LicensePool, and a redundant 'license add' event
-        # which was registered with analytics but which did not
-        # affect the counts.
-        assert 4 == analytics.count
+        # Three analytics events were collected: one for the license
+        # add event itself, one for the 'checkin' that made the new
+        # license available, and a redundant 'license add' event which
+        # was registered with analytics but which did not affect the
+        # counts.
```

Comment:
I still don't understand this redundant event but it's not new in this branch so I didn't investigate too closely.